### DISCUSSION TITLE
Welcome to ksctl Discussions!

### DISCUSSION THREAD

**Discussion by dipankardas011** [kubesimplify/ksctl#52] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rahul-Kumar-prog** [comment]
Yeah so Hey @dipankardas011 My name is Rahul, so i would like to work on #129 . but first i have to go through with ksctl for understanding. 
so should i take it or should i learn ksctl first. or is there any work i can do as a beginner ?

**Reply by dipankardas011** [reply]
I'm sorry but the issue is not suitable for beginners and requires thorough planning. However, once you have a good understanding of the ksctl project, you can consider working on it. Right now, we are actively working on making major changes in progress #123

**Reply by Rahul-Kumar-prog** [reply]
ok I understand. Thankyou
